scipy/sparse/bsr.py

```diff
@@ -407,14 +407,14 @@ def tobsr(self,blocksize=None,copy=False):
         else:
             return self
 
-    def tocsr(self):
-        return self.tocoo(copy=False).tocsr()
+    def tocsr(self, copy=False):
+        return self.tocoo(copy=copy).tocsr()
```

Comment:
The latter `tocsr` always makes a copy, so it seems to me the call to `tocoo` can always have `copy=False` as it did before?